File: legacy_samples/legacy.java
public class Main {
    public static int add(int a, int b) {
        return a + b;
    }

    public static String greet(String name) {
        return "Hello, " + name + "!";
    }

    public static void main(String[] args) {
        System.out.println(greet("world"));
        System.out.println(add(2, 3));
    }
}

File: legacy_samples/legacy.py
import os


def greet(name):
    return "Hello, %s!" % name


def read_config_path(base_dir, filename):
    return os.path.join(base_dir, filename)


def main():
    print(greet("world"))
    print(read_config_path("/etc", "config.ini"))


if __name__ == "__main__":
    main()
